src/raftstore/store/worker/apply.rs

```diff
@@ -1106,40 +1210,62 @@ impl Runner {
         }
     }
 
-    fn handle_apply(&mut self, apply: Apply) {
-        if apply.entries.is_empty() {
-            return;
-        }
-        let mut e = match self.delegates.entry(apply.region_id) {
-            MapEntry::Vacant(_) => {
-                error!("[region {}] is missing", apply.region_id);
-                return;
-            }
-            MapEntry::Occupied(e) => e,
-        };
-        {
-            let delegate = e.get_mut();
-            delegate.metrics = ApplyMetrics::default();
-            delegate.term = apply.term;
-            let results = delegate.handle_raft_committed_entries(apply.entries);
+    fn handle_apply(&mut self, applys: Vec<Apply>) {
+        let timer = STORE_APPLY_LOG_HISTOGRAM.start_timer();
```

Comment:
you can use `let _timer`, so you don't need to call `observe_duration` explicitly in the end of this function.